### PR TITLE
Add class for Avatars

### PR DESCRIPTION
**Pull request by bzf**
* Fix padding from items in a DropdownButton
* Fix padding in Modal for mobile
* Extract padding for content-list-item to variable

### CODE REVIEW

File: src/mnd-bootstrap/components/_avatar.scss [hunk 1]
```diff
@@ -0,0 +1,79 @@
+/*doc
```
Comment (mnd-hound) [inline src/mnd-bootstrap/components/_avatar.scss]: Use `//` comments everywhere

File: src/mnd-bootstrap/components/_avatar.scss [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+<div class="avatar">
+  <div class="avatar-name">
+    A
+  </div>
+</div>
+```
+*/
+
+@import 'mnd-bootstrap/variables/colors';
```
Comment (mnd-hound) [inline src/mnd-bootstrap/components/_avatar.scss]: Prefer double-quoted strings

File: src/mnd-bootstrap/components/_avatar.scss [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+@import 'mnd-bootstrap/variables/measures';
```
Comment (mnd-hound) [inline src/mnd-bootstrap/components/_avatar.scss]: Prefer double-quoted strings

File: src/mnd-bootstrap/components/_content_list.scss [hunk 4]
```diff
@@ -103,7 +105,7 @@ Custom list for editable content
 
   .content-list-header {
     color: $black;
-    font-weight: 600;
+    font-weight: $semi-bold;
```
Comment (mnd-hound) [inline src/mnd-bootstrap/components/_content_list.scss]: Properties should be ordered color, font-size, font-weight

**Comment by AndreasArledal** [comment]
:+1:

**Pull request merged**